csrankings.csv

```diff
@@ -441,6 +441,7 @@ Alberto H. F. Laender,UFMG,http://homepages.dcc.ufmg.br/~laender,HJhu4csAAAAJ
 Alberto L. Sangiovanni-Vincentelli,University of California - Berkeley,https://www2.eecs.berkeley.edu/Faculty/Homepages/sangiovanni-vicentelli.html,AhgjQ2QAAAAJ
 Alberto Lluch Lafuente,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Alberto Manuel Rodrigues da Silva,Universidade de Lisboa,http://isg.inesc-id.pt/alb,NjZqNwEAAAAJ
+Alberto Marchetti-Spaccamela,Sapienza University of Rome,http://www.diag.uniroma1.it/users/alberto marchetti-spaccamela,xITAf6EAAAAJ

```

Comment:
Incorrect URL - should be http://www.dis.uniroma1.it/~alberto/.

Reply:
Thank you very much for finding the issue with the whitespace symbol! Just replaced with the URL-encoded character: http://www.diag.uniroma1.it/users/alberto%20marchetti-spaccamela
The URL you suggest is correct but, unfortunately, outdated. Thanks again!